gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -183,6 +232,22 @@ def main(args):
         )
         count_ht.write(res.vep_count_ht.path, overwrite=args.overwrite)
 
+    if args.generate_trio_stats:
+        res = resources.generate_trio_stats
```

Comment:
outside the scope of this PR, but may be a bit confusing to have a 'generate_trio_stats' function, a 'generate_trio_stats' attribute, and a 'generate_trio_stats' argument.

Reply:
ah okay I see that way of thinking about it - makes sense!